internal/cli/mongocli/serverless/backup/backup.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 MongoDB Inc
```

Comment:
I don't think we need this entire folder, double check if you duplicated other files without need.

Reply:
A bit easier to review if looking specifically at the [Recreate serverless commands for MongoCLI commit](https://github.com/mongodb/mongodb-atlas-cli/pull/1841/commits/969a5486c09ed3c67698522bb889eb4fe5355c3c#diff-63567b515afffd64b3bbef2fce4778433d3c713843e7fe2e10f11a724d23cd7f), but previously we had a Builder for [Atlas CLI](https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/cli/root/atlas/builder.go#L215) `AtlasCLIBuilder()` and the builder [for MongoCLI](https://github.com/mongodb/mongodb-atlas-cli/blob/6a4bc44c8210b78f10e8d62e01dbb7fad3b00587/internal/cli/atlas/atlas.go#L87) `Builder()`

TL;DR: `AtlasBuilder()` is used by AtlasCLI and has some additional commands compared to `Builder()`, which is used by MongoCLI, hence the split

Reply:
Everything that's duplicated previously had two versions of the Builders, one for AtlasCLI and one for MongoCLI, so each of those has been split and the callers updated with the appropriate references